File: pigatize.rb
class Pigatize

  def self.pigatize(text)
    vowels = ['a','e','i','o','u']
    word = text.slice(0)
    if vowels.include? word[0]
      text << 'way'
    else
      text << word << 'ay'
    end
  end
end

puts Pigatize.pigatize('tacos')

File: pigatize.js
vowel = ['a','e','i','o','u'];

function pigatize(text) {
    var word = text.slice(0,1);
    if (word.includes('vowel')) {
        output = text + 'way';
    }
    else {
       output = text + word + 'ay';
    }
    return output;
}

console.log(pigatize('apple'));